primitives/runtime/src/bounded/bounded_btree_map.rs

```diff
@@ -161,6 +161,39 @@ where
 	{
 		self.0.remove_entry(key)
 	}
+
+	/// Consume the map, applying `f` to each of it's values and returning a new map.
+	pub fn map<T, F>(self, mut f: F) -> BoundedBTreeMap<K, T, S>
+	where
+		F: FnMut((&K, V)) -> T,
+	{
+		self.0
+			.into_iter()
+			.map(|(k, v)| {
+				let t = f((&k, v));
+				(k, t)
+			})
+			.try_collect()
```

Comment:
Why `try_collect` and `unwrap()` when you can just `collect`?

Reply:
Good point, fixed in https://github.com/paritytech/substrate/pull/11869/commits/2a993a48d47148d3f7db2ce577c7d293f00fd76b